test/functional/cmdLineTests/shareClassTests/DataHelperTests/DataHelperTests.xml

```diff
@@ -54,17 +54,14 @@
 	<if testVariable="SCMODE" testValue="213" resultVariable="currentMode" resultValue="$mode213$"/>
 	<if testVariable="SCMODE" testValue="214" resultVariable="currentMode" resultValue="$mode214$"/>
 
-	<echo value=" "/>
+	<echo value=""/>
```

Comment:
Apparently `com.oti.j9.exclude.XMLParser` doesn't handle empty strings: please keep the spaces in places like this for now.

Reply:
On second thought, let's keep the empty strings: when https://github.com/eclipse-openj9/openj9/pull/20768 is merged, they'll be supported.